src/modules/clusters/import-cluster/import-cluster.js

```diff
@@ -80,7 +111,7 @@
          */
         function viewTaskProgress() {
 
-           if (vm.clusterId) {
+            if (vm.clusterId) {
```

Comment:
@a2batic Please correct the indentation here.

Reply:
Acknowledge

Reply:
@a2batic ok. Thanks.